gobblin-dataset-config/src/main/java/gobblin/dataset/config/DatasetUtils.java

```diff
@@ -0,0 +1,80 @@
+package gobblin.dataset.config;
+
+import java.util.*;
+
+
+public class DatasetUtils {
+  public static final String ID_DELEMETER = ".";
+  public static final String DATASETS_PREFIX = "config-ds";
+  public static final String TAGS_PREFIX = "config-tag";
+  public static final String CONSTS_PREFIX = "config-const";
+  public static final String ROOT = "root";
+  public static final String IMPORTED_TAGS = "imported-tags";
+
+  public static final String getParentId(String urn) {
```

Comment:
I thought we agreed on using URIs of the form: <config_store_scheme>://[<config_store_server>]:/config/path.